src/lib/storage/table.cpp

```diff
@@ -193,6 +194,51 @@ void Table::append_chunk(const std::shared_ptr<Chunk>& chunk) {
   _chunks.push_back(chunk);
 }
 
+std::vector<AllTypeVariant> Table::get_row(size_t row_idx) const {
+  for (const auto& chunk : _chunks) {
```

Comment:
Performance warnings for both new methods? They look by far too nice to be not used by students.

Reply:
yesss